app/experimenter/experiments/models.py

```diff
@@ -1026,13 +1026,19 @@ def latest_change(self):
     def latest_review_request(self):
         return (
             self.all()
-            .filter(NimbusChangeLog.Filters.IS_REVIEW_REQUEST)
+            .filter(
+                NimbusChangeLog.Filters.IS_REVIEW_REQUEST
+                | NimbusChangeLog.Filters.IS_UPDATE_REVIEW_REQUEST
+            )
             .order_by("-changed_on")
         ).first()
 
     def latest_rejection(self):
         change = self.latest_change()
-        if change and change.has_filter(NimbusChangeLog.Filters.IS_REJECTION):
+        if change and change.has_filter(
+            NimbusChangeLog.Filters.IS_REJECTION
+            | NimbusChangeLog.Filters.IS_UPDATE_REJECTION
```

Comment:
Actually we could do this another way and have like

```py
IS_LAUNCH_REJECTION = ....
IS_UPDATE_REJECTION = ....
IS_REJECTION = IS_LAUNCH_REJECTION | IS_UDPATE_REJECTION
```

and then only use `IS_REJECTION` in these methods.  Similar for `IS_REVIEW_REQUEST`.  I guess it depends on how/where we want to treat review/rejection cases the same/different.  We can also just keep an eye on it and move things around later if we want.